File: app/MyJobPitch/app/src/main/java/com/myjobpitch/api/data/Business.java
package com.myjobpitch.api.data;

import com.myjobpitch.api.MJPObjectWithDates;

import java.util.List;

public class Business extends MJPObjectWithDates {
    private String name;
    private List<Integer> users;
    private List<Integer> locations;
    private List<Image> images;
    private Integer  tokens;

    public String getName() {
        return name;
    }

    public List<Integer> getUsers() {
        return users;
    }

    public List<Integer> getLocations() {
        return locations;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Image> getImages() {
        return images;
    }

    public Integer getTokens() {
        return tokens;
    }
}

File: ios/MyJobPitch/Models/api/Business.swift
//
//  Business.swift
//  MyJobPitch
//
//  Created by dev on 12/20/16.
//  Copyright © 2016 myjobpitch. All rights reserved.
//

import Foundation

class Business: MJPObjectWithDates {

    var name: String!
    var users: NSArray!
    var locations: NSArray!
    var images: NSArray!
    var tokens: NSNumber!
    
    // API V5
    var restricted = false
    
    func getImage() -> Image? {
        if images != nil && images.count > 0 {
            return images.firstObject as? Image
        }
        return nil
    }
    
}
